numpy/_core/defchararray.py

```diff
@@ -995,6 +1077,15 @@ def istitle(a):
     See Also
     --------
     str.istitle
+
+    Examples
+    --------
+    >>> np.char.istitle("Ghc")
```

Comment:
Maybe also add an example that's got a second word as well to better demonstrate what title-case is.